src/app/shared/services/claims/claims.service.ts

```diff
@@ -47,14 +52,28 @@ export class ClaimsService {
       this.iamService.claimsService.getClaimsByRequester({
         did: this.iamService.signerService.did,
       })
+    ).pipe(
+      catchError((err) => {
+        console.error(err);
+        return of(err);
+      }),
+      finalize(() => {
+        if (showLoader) {
+          this.loadingService.hide();
+        }
+      })
```

Comment:
Is the key change that `finalize` with the `loadingService.hide()` was missing? Or more that the `catchError` was missing?

Reply:
No, i did not hide it. So the key is `finalize`. However i can't use it without `catchError`. It's like try-catch-finally.

Reply:
I though catchError is needed, but it looks like it's not.

Reply:
Ok, thanks!